scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -634,8 +644,8 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     ts.toList
   }
 
-  @inline final def commaSeparated[T <: Tree : AstMetadata](part: => T): List[T] =
-    tokenSeparated[`,`, T](sepFirst = false, part)
+  @inline final def commaSeparated[T <: Tree : AstMetadata](part: => T, useUnquoteForEllipsis: Boolean = false): List[T] =
+    tokenSeparated[`,`, T](sepFirst = false, part, useUnquoteForEllipsis)
```

Comment:
Usually ellipsis is parsed with `part`. But sometimes (for example, when parsing paramClauses) it should simple be `ellipsis(1, unquote[...])`, because `part` works ambiguously.